docs/vNext/DeclarativeWorkloadModule.md

```diff
@@ -0,0 +1,196 @@
+---
+layout: vNext
+title:  "Declarative Workloads"
+categories: reference
+permalink: /vNext/declarative-workload-module/
+order: 4
+---
+## Table of contents
+
+- [Overview](#overview)
+- [Example](#example)
+- [Hierarchy](#hierarchy)
+  - [Contracts](#contracts)
+  - [Functions](#functions)
+  - [Parameters](#parameters)
+    - [Uniform Random](#uniform-random)
+    - [Parameters Reference](#parameter-reference)
+    - [Variable Reference](#variable-reference)
+    - [List Element](#list-element)
+    - [Formatted String](#formatted-string)
+- [Implementing the DeclarativeWorkloadModuleBase class](#implementing-the-declarativeworkloadmodulebase-class)
+  - [submitWithArguments](#submitwitharguments)
+## Overview
+`DeclarativeWorkloadModuleBase` is a base class that can be implemented for declaratively assigning workload parameters. The Contracts, Functions and Parameters for test runs are specified under `arguments.behavior`.
+
+## Example
+
+```yaml
+workload:
+  module: declarative
+    arguments:
```

Comment:
The indentation is still wrong, `arguments` should be on the same level as `module`.